src/clr-core/.storybook/addons.js

```diff
@@ -0,0 +1,25 @@
+import { addons } from '@storybook/addons';
+import '@storybook/addon-docs/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-actions/register';
+import 'storybook-addon-designs/register';
+import '@storybook/addon-a11y/register';
+import '@storybook/addon-cssresources/register';
+import { create } from '@storybook/theming/create';
+
+addons.setConfig({
+  options: { showPanel: true },
+  theme: create({
+    base: 'light',
+    brandTitle: 'Clarity Design - Web Components',
+    brandUrl: 'https://clarity.design',
+    brandImage: './assets/images/clarity-logo.svg',
+  }),
+});
+
+setTimeout(() => {
```

Comment:
LOL. `setTimeout` is like the American Express of JavaScript ("it's everywhere you wanna be!")

Reply:
Yeah that was the workaround in their github issue :/